lib/features.js

```diff
@@ -5,6 +5,9 @@
 'use strict'
 
 const crypto = require('crypto')
+const isA = require('joi')
+
+const SCHEMA = isA.array().items(isA.string()).optional()
```

Comment:
I don't think it's worth the maintenance burden of having to come and change this module each time we add/remove/edit a feature. Plus it introduces the possibility of forgetting to update it here.